beacon/blockchain/payload.go

```diff
@@ -61,7 +68,7 @@ func (s *Service) forceSyncUponProcess(
 			SafeBlockHash:      lph.GetParentHash(),
 			FinalizedBlockHash: lph.GetParentHash(),
 		},
-		s.chainSpec.ActiveForkVersionForTimestamp(lph.GetTimestamp()),
+		forkVersion,
```

Comment:
just noting that this change likely does not matter as this fork version is only used to validate that we are **not before Deneb** [here](https://github.com/berachain/beacon-kit/blob/main/execution/client/ethclient/engine.go#L121-L124). So if this is Deneb, Deneb1, Electra, it will not error on our side.

Reply:
Also will not affect the FCU call to the EL either

Reply:
Totally agree. Should we then drop the fork check?
It is very weirdly placed to me, considering EL is not aware of CL forks.
Also we may just move it to relevant `ValidateAfterDecodingSSZ` or to other constructors.
cc @rezbera, @fridrik01